security-annotations/build.gradle.kts

```diff
@@ -0,0 +1,6 @@
+plugins {
+    id("io.micronaut.build.internal.security-module")
+}
+dependencies {
+    implementation("io.micronaut:micronaut-core-processor")
```

Comment:
should  be `compileOnly` because this module is transitively used by the runtime modules

Reply:
changed to `compileOnly`